src/Screens/AllEventsScreen.js

```diff
@@ -26,11 +25,13 @@ let AllEventsScreen = ({allEvents}) => {
           {
```

Comment:
this is great but this is what I have in my branch right now:

`{
            allEvents.map(event => {
              return (
                  <Link to={`/events/${event.eventid}`}>
                    <p>{event.eventTitle}</p>
                    <p>{event.Date}</p>
                    <div>
                      <p>{event.mealType}</p>
                      <p>{event.eventSize}</p>
                      <img src={shoppingListIcon} alt="shopping-list-icon" style={style}/>
                    </div>
                  </Link>
              )
            }
            )
          }`
Can we hold off on your changes until my branch is done?